docs/self_hosted/optional_features.md

```diff
@@ -0,0 +1,31 @@
+---
+id: optional_features
+title: Optional features
+sidebar_label: Optional features
+---
+
+Mia-Platform Console offers some optional features which are disabled by default and can be enabled as needed.
+
+Optional features can be enabled using **feature toggles** by a Console Super User.
+
+Features have one or multiple scopes, which are indicative of the Console resource type or the user interested by that feature. Available scope types are: Console, Company, Project, Environment, and User.
+
+*Console* scope means that a feature is valid for the whole Console instance; all the other scopes require to provide a scope ID.
+
+Available features are visible from the *Features* section of the CMS.
+
+To enable a feature, navigate to the *Feature Activations* section of the CMS, which shows a table. Each entry in the table in *PUBLIC* state represents an active feature.
+
+Click "Add new" and enter the feature ID, the scope type, and the scope ID (leave this last field empty in case of Console scope type). The just added feature activation will be in *PUBLIC* state by default and immediately enabled.
+
+### Google Analytics Tracking
```

Comment:
This is not managed also by the helm chart?

Reply:
this has been inserted in console version upgrader; it is among features.js, but it is not in features activation file